recipe/0068-newspaper/newspaper_title-collection.json

```diff
@@ -0,0 +1,96 @@
+{
+  "@context": [
+    "http://iiif.io/api/presentation/3/context.json"
+  ],
+  "type": "Collection",
+  "id": "{{ id.url }}",
+  "within": "http://data.theeuropeanlibrary.org/Collection/a0620",
+  "label": {
+    "de": [
+      "Berliner Tageblatt"
+    ]
+  },
+  "metadata": [
+    {
+      "label": {
+        "en": [
+          "type"
+        ]
+      },
+      "value": {
+        "none": "http://data.europeana.eu/concept/base/18",
+        "en": [
+          "Newspaper Title",
+          "Serial"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "language"
+        ]
+      },
+      "value": {
+        "none": "de",
```

Comment:
We think this should be an array e.g.:
"none": ["de"]

Reply:
done